packages/kbn-expandable-flyout/src/context.tsx

```diff
@@ -59,19 +67,40 @@ export const ExpandableFlyoutContext = createContext<ExpandableFlyoutContext | u
   undefined
 );
 
+export type ExpandableFlyoutApi = Pick<ExpandableFlyoutContext, 'openFlyout'> & {
+  getState: () => State;
+};
+
 export interface ExpandableFlyoutProviderProps {
   /**
    * React children
    */
   children: React.ReactNode;
+
+  onChanges?: (payload: any) => void;
```

Comment:
do you mind adding a brief description here and the line below? just for consistency with the rest of the package

Reply:
sure will do!